fmriprep/workflows/util.py

```diff
@@ -233,13 +258,13 @@ def init_bbreg_wf(bold2t1w_dof, name='bbreg_wf'):
             't1_seg', 't1_brain']),  # FLIRT BBR
         name='inputnode')
     outputnode = pe.Node(
-        niu.IdentityInterface(['itk_bold_to_t1', 'itk_t1_to_bold', 'out_report']),
+        niu.IdentityInterface(['itk_bold_to_t1', 'itk_t1_to_bold', 'out_report', 'fallback']),
         name='outputnode')
 
-    bbregister = pe.Node(
-        BBRegisterRPT(dof=bold2t1w_dof, contrast_type='t2', init='coreg',
-                      registered_file=True, out_lta_file=True, generate_report=True),
-        name='bbregister')
+    mri_coreg = pe.Node(
```

Comment:
Does MRICoreg generate a new T1w space with the appropriate xforms? I'll need to get this clear before I start with #745.

Reply:
No. `mri_coreg` only generates a transform matrix. If you want to apply the transform, you can use `mri_vol2vol` (see [MRICoregRPT](https://github.com/poldracklab/niworkflows/blob/c55052435622efdc57959702af51178a4d774fae/niworkflows/interfaces/registration.py#L231-L234) or `mri_convert`).